src/enemy.c

```diff
@@ -364,7 +309,7 @@ float enemy_get_hurt_radius(Enemy *enemy) {
 
 static bool should_auto_kill(Enemy *enemy) {
 	return
-		(enemy->hp <= 0 && enemy->hp != _internal_ENEMY_IMMUNE) ||
```

Comment:
why is this no longer needed?

Reply:
It's not used anymore. The same effect can be achieved by adding `EFLAG_NO_AUTOKILL` to `enemy->flags`. `EFLAGS_GHOST` is a set of flags that generally matches the old behavior of `ENEMY_IMMUNE`.